projects/ng-core-tester/src/app/app-routing.module.ts

```diff
@@ -309,9 +309,33 @@ const routes: Routes = [
             }
           ],
           searchFilters: [
+            {
+              label: 'Advanced search',
+              filter: 'simple',
+              value: '0',
+              /* If you set this value, the url parameter will still be present,
+              but with different values */
+              disabledValue: '1',
+              url: {
+                routerLink: ['/record', 'search', 'documents'],
+                title: 'Link to search document'
+              }
+            },
```

Comment:
If I understand : an advanced search url should be : `http://localhost/record/search/documents?simple=1` ?
For me, this url seems strange :(

Reply:
advanced search url is `http://localhost/record/search/documents?simple=0`